vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -181,7 +203,7 @@ func (r RefCountsMap) Init(d drivers.VolumeDriver, mountDir string, name string)
 	err = r.discoverAndSync(c, d)
```

Comment:
Failing the mount requests means containers won't start after the plugin has ack'ed that its done init'ing. Unmounts can't be failed because then we will be missing unmount refcounts. Still needs code to handle refcounts that may be getting restored for the very same containers by the refcount-recalc thread.

This is different from what we discussed offline.  Frankly we need two bitmaps - an active one and one to handle collecting refcounts for containers that started before the plugin restarted.